dspace/config/emails/bte_batch_import_error

```diff
@@ -0,0 +1,16 @@
+## Email sent to DSpace users when they BTE batch import fails.
```

Comment:
This email appears to be used [here](https://github.com/TAMULib/DSpace/blob/tamu-dspace-7_x/dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java#L2252).